### PR TITLE
(feat)chaos-charts: adding chaosengine yaml

### PR DESCRIPTION
**Pull request by aditya109**
Signed-off-by: aditya109 <adikid1996@gmail.com>

- added `engine.yaml` for all the experiments

### CODE REVIEW

File: charts/coredns/coredns-pod-delete/engine.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  name: engine-coredns
+  namespace: kube-system
+spec:
+  appinfo:
+    appns: kube-system
+    applabel: 'k8s-app=kube-dns'
+    appkind: deployment
+  # It can be infra only
+  chaosType: 'infra'
+  #ex. values: ns1:name=percona,ns2:run=nginx
```
Comment (amitbhatt818) [inline charts/coredns/coredns-pod-delete/engine.yaml]: Remove chaosType and add annotationCheck

Comment (amitbhatt818) [reply charts/coredns/coredns-pod-delete/engine.yaml]: Refer this PR: https://github.com/litmuschaos/litmus-docs/pull/182

File: charts/coredns/coredns-pod-delete/engine.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    applabel: 'k8s-app=kube-dns'
+    appkind: deployment
+  # It can be infra only
+  chaosType: 'infra'
+  #ex. values: ns1:name=percona,ns2:run=nginx 
+  auxiliaryAppInfo: ""
+  components:
+    runner:
+       image: "litmuschaos/chaos-executor:1.0.0"
+       type: "go"
```
Comment (amitbhatt818) [inline charts/coredns/coredns-pod-delete/engine.yaml]: Replace double quotes to single quotes in each file

**Review by amitbhatt818 — CHANGES_REQUESTED** [review]

File: charts/coredns/coredns-pod-delete/engine.yaml (hunk 2 above)
Comment (aditya109) [reply charts/coredns/coredns-pod-delete/engine.yaml]: But it is in docs. I have them up from there.

Comment (aditya109) [reply charts/coredns/coredns-pod-delete/engine.yaml]: I have picked these yamls from docs, so should I not pick them from there??

**Pull request closed without merging**